index.js

```diff
@@ -124,27 +122,23 @@ io.on("connection", socket => {
 		socket.to(roomID).emit("playerHandler", ReturnPlayerReady); 
 		socket.emit("playerHandler", ReturnPlayerReady);
 	});
-	/*socket.on("testEvent", () => {
-		socket.join("/123456");
-		console.log("User joined ", socket.id);
-	})*/
 
 	//Listens for a 'startGame' event and either emits a 'startedGame' event to all clients in a room if conditions are met, or sends a 'cantStartGame' event to the initiating client if not.
 	socket.on("startGame", () => {
 		const roomID = PlayerRooms.get(socket.id);
 		if(ShouldStartGame(roomID)) {
 			const roomData = Rooms.get(roomID);
-			
-			socket.to(roomID).emit("startedGame");
-			socket.emit("startedGame");
+			const startedGameData = {lives: roomData.settings.life,handSize:roomData.settings.handSize}
```

Comment:
Ku være lækkert at have json objekter til at se noget ala det her ud, således man faktisk kan læse det :)

```suggestion
			const startedGameData = {
				lives: roomData.settings.life,
				handSize: roomData.settings.handSize
			}
```